### PR TITLE
Parachain upgraded to 0.9.32

### PR DESCRIPTION
**Pull request by zktony**
Parachain upgraded to 0.9.32

### CODE REVIEW

File: runtime/src/xcm_config.rs [hunk 1]
```diff
@@ -60,43 +60,111 @@ pub type XcmOriginToTransactDispatchOrigin = (
...
+		message: &mut Xcm<RuntimeCall>,
+		max_weight: XCMWeight,
+		weight_credit: &mut XCMWeight,
+	) -> Result<(), ()> {
+		Deny::should_execute(origin, message, max_weight, weight_credit)?;
+		Allow::should_execute(origin, message, max_weight, weight_credit)
+	}
+}
+
+// See issue #5233
```
Comment (serhii-temchenko) [inline runtime/src/xcm_config.rs:114]: Where this issue can be found?

**Review by serhii-temchenko — CHANGES_REQUESTED** [review]

File: node/Cargo.toml [hunk 2]
```diff
@@ -9,84 +9,77 @@ repository = "https://github.com/paritytech/cumulus/"
...
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.32" }
+sp-keystore = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.32" }
+sp-offchain = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.32" }
+sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.32" }
+sp-session = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.32" }
+sp-timestamp = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.32" }
+sp-transaction-pool = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.32" }
+substrate-frame-rpc-system = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.32" }
+substrate-prometheus-endpoint = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.32" }
+try-runtime-cli = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.32" }
```
[the remark this answers is not in the document]
Comment (serhii-temchenko) [reply node/Cargo.toml:55]: Found out that this wouldn't work as I assumed. If check [this](https://rust-lang.github.io/rfcs/2906-cargo-workspace-deduplicate.html#new-dependency-directives) section - it says:

> For now if a `workspace = true` dependency is specified then also specifying the `default-features` value is disallowed. The `default-features` value for a directive is inherited from the `[workspace.dependencies]` declaration, which defaults to true if nothing else is specified.

Plus I tried locally as well and received some circular deps error on build... But if I get it right - then we need to remove `default-feature` from the `runtime` crate [here](https://github.com/Polkadex-Substrate/parachain/blob/3621afaed8afcf378f69a89b7a78dc40506f6ef1/runtime/Cargo.toml#L18-L71)?
I'm about:

> ...specifying the `default-features` value is disallowed. ...

File: node/src/chain_spec.rs [hunk 3]
```diff
@@ -239,23 +186,24 @@ fn testnet_genesis(
 	endowed_accounts: Vec<AccountId>,
 	id: ParaId,
 	root_key: AccountId
-) -> parachain_template_runtime::GenesisConfig {
-	parachain_template_runtime::GenesisConfig {
-		system: parachain_template_runtime::SystemConfig {
-			code: parachain_template_runtime::WASM_BINARY
+) -> parachain_polkadex_runtime::GenesisConfig {
+	let alice = get_account_id_from_seed::<sr25519::Public>("Alice");
```
[the remark this answers is not in the document]
Comment (zktony) [reply node/src/chain_spec.rs:190]: Yeah .. I removed all warnings in #3

File: runtime/src/xcm_config.rs (hunk 1 above)
Comment (serhii-temchenko) [reply runtime/src/xcm_config.rs:114]: Ok, I can see that this is a copy/paste from a different repo and used only for testing purpose but would be nice to have such things cleaned up...

**Review by serhii-temchenko — APPROVED** [review]

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**